src/main/java/com/redhat/devtools/intellij/tektoncd/tree/TasksNode.java

```diff
@@ -26,7 +26,7 @@ public void load() {
         super.load();
         try {
             NamespaceNode namespaceNode = (NamespaceNode) getParent();
-            ((TektonRootNode)getRoot()).getTkn().getTasks(namespaceNode.toString()).forEach(task -> add(new TaskNode(task)));
+            ((TektonRootNode)getRoot()).getTkn().getTasks(namespaceNode.toString()).forEach(task -> { if(!task.isEmpty()) add(new TaskNode(task)); });
```

Comment:
I think this should be done at the CLI level, the result of getTasks should not return empty names !!!. Using Java Streams could filter the list